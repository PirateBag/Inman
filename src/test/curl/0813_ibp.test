Report on running inventory balanhce for all items.
=== JSON ===
{ "idToSearchFor" : "-1" }
