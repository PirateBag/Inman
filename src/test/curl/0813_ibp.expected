{
  "responseType" : "MULTILINE",
  "errors" : [ ],
  "data" : [ {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Inventory Analysis for 0001: W-001,36 In Red Wagon     0.00 MAN   0   2     5.00     1.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Opening Balance of 5.0 and there are 1 open orders"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  Id Item ParentId  Ordered Assigned      Start   Complete Stat   Type Acti  Balance"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  27    1        0   500.00     0.00  2025-0913  2025-0915 OPEN MOHEAD NONE   505.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Inventory Analysis for 0002: W-002,Red Wagon Body     0.00 MAN   1  10     0.00     1.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Opening Balance of 0.0 and there are 2 open orders"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  Id Item ParentId  Ordered Assigned      Start   Complete Stat   Type Acti  Balance"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  35    2        0   500.00     0.00  2025-0901  2025-0911 OPEN MOHEAD NONE   500.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  28    2       27  -500.00     0.00  2025-0911  2025-0913 OPEN  MODET NONE     0.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Inventory Analysis for 0003: W-003,Front Wheel Asmbly     0.00 MAN   1   1     0.00     1.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Opening Balance of 0.0 and there are 3 open orders"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  Id Item ParentId  Ordered Assigned      Start   Complete Stat   Type Acti  Balance"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  38    3        0   500.00     0.00  2025-0910  2025-0911 OPEN MOHEAD NONE   500.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  29    3       27  -500.00     0.00  2025-0911  2025-0913 OPEN  MODET NONE     0.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  15    3        0   500.00     0.00  2025-0914  2025-0915 OPEN MOHEAD NONE   500.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Inventory Analysis for 0004: W-004,Rear Wheel Asmbly     0.00 MAN   1   1     0.00     1.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Opening Balance of 0.0 and there are 2 open orders"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  Id Item ParentId  Ordered Assigned      Start   Complete Stat   Type Acti  Balance"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  43    4        0   500.00     0.00  2025-0910  2025-0911 OPEN MOHEAD NONE   500.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  30    4       27  -500.00     0.00  2025-0911  2025-0913 OPEN  MODET NONE     0.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Inventory Analysis for 0006: W-006,Decal     0.25 PUR   1   5     0.00     1.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Opening Balance of 0.0 and there are 2 open orders"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  Id Item ParentId  Ordered Assigned      Start   Complete Stat   Type Acti  Balance"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  51    6        0  1000.00     0.00  2025-0906  2025-0911 OPEN     PO NONE  1000.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  31    6       27 -1000.00     0.00  2025-0911  2025-0913 OPEN  MODET NONE     0.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Inventory Analysis for 0008: W-008,Threaded bolt     0.60 PUR   1   5     0.00   500.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Opening Balance of 0.0 and there are 2 open orders"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  Id Item ParentId  Ordered Assigned      Start   Complete Stat   Type Acti  Balance"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  52    8        0  2000.00     0.00  2025-0906  2025-0911 OPEN     PO NONE  2000.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  33    8       27 -2000.00     0.00  2025-0911  2025-0913 OPEN  MODET NONE     0.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Inventory Analysis for 0005: W-005,Axle     0.00 MAN   2   4     0.00     1.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Opening Balance of 0.0 and there are 10 open orders"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  Id Item ParentId  Ordered Assigned      Start   Complete Stat   Type Acti  Balance"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  53    5        0  1000.00     0.00  2025-0905  2025-0909 OPEN MOHEAD NONE  1000.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  39    5       38  -500.00     0.00  2025-0909  2025-0910 OPEN  MODET NONE   500.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  44    5       43  -500.00     0.00  2025-0909  2025-0910 OPEN  MODET NONE     0.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "   1    5        0     6.00     0.00  2025-0901  2025-0913 OPEN MOHEAD NONE     6.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  20    5        0   494.00     0.00  2025-0909  2025-0913 OPEN MOHEAD NONE   500.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  16    5       15  -500.00     0.00  2025-0913  2025-0914 OPEN  MODET NONE     0.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "   4    5        0   500.00     0.00  2025-0911  2025-0915 OPEN MOHEAD NONE   500.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "   6    5        0  1000.00     0.00  2025-1011  2025-1015 OPEN MOHEAD NONE  1500.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "   8    5        0  1500.00     0.00  2025-1111  2025-1115 OPEN MOHEAD NONE  3000.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  10    5        0    50.00     0.00  2025-1211  2025-1215 OPEN MOHEAD NONE  3050.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Inventory Analysis for 0007: W-007,Washer     0.21 PUR   2   5     0.00  1000.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Opening Balance of 0.0 and there are 4 open orders"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  Id Item ParentId  Ordered Assigned      Start   Complete Stat   Type Acti  Balance"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  55    7        0  1000.00     0.00  2025-0904  2025-0909 OPEN     PO NONE  1000.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  45    7       43 -1000.00     0.00  2025-0909  2025-0910 OPEN  MODET NONE     0.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  56    7        0  4000.00     0.00  2025-0906  2025-0911 OPEN     PO NONE  4000.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  32    7       27 -4000.00     0.00  2025-0911  2025-0913 OPEN  MODET NONE     0.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Inventory Analysis for 0009: W-009,Nut     0.17 PUR   2   6     0.00   500.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Opening Balance of 0.0 and there are 8 open orders"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  Id Item ParentId  Ordered Assigned      Start   Complete Stat   Type Acti  Balance"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  57    9        0  2000.00     0.00  2025-0903  2025-0909 OPEN     PO NONE  2000.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  40    9       38 -1000.00     0.00  2025-0909  2025-0910 OPEN  MODET NONE  1000.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  46    9       43 -1000.00     0.00  2025-0909  2025-0910 OPEN  MODET NONE     0.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  58    9        0  1000.00     0.00  2025-0905  2025-0911 OPEN     PO NONE  1000.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  22    9        0  1000.00     0.00  2025-0907  2025-0913 OPEN     PO NONE  2000.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  34    9       27 -2000.00     0.00  2025-0911  2025-0913 OPEN  MODET NONE     0.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  59    9        0  1000.00     0.00  2025-0907  2025-0913 OPEN     PO NONE  1000.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  17    9       15 -1000.00     0.00  2025-0913  2025-0914 OPEN  MODET NONE     0.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Inventory Analysis for 0010: W-010,8 In Wheel     0.17 PUR   2  10     0.00   120.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Opening Balance of 0.0 and there are 2 open orders"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  Id Item ParentId  Ordered Assigned      Start   Complete Stat   Type Acti  Balance"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  60   10        0  1000.00     0.00  2025-0830  2025-0909 OPEN     PO NONE  1000.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  47   10       43 -1000.00     0.00  2025-0909  2025-0910 OPEN  MODET NONE     0.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Inventory Analysis for 0011: W-011,Front Wheel Bracket     0.95 PUR   2   2     0.00    50.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Opening Balance of 0.0 and there are 4 open orders"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  Id Item ParentId  Ordered Assigned      Start   Complete Stat   Type Acti  Balance"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  61   11        0  1000.00     0.00  2025-0907  2025-0909 OPEN     PO NONE  1000.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  41   11       38 -1000.00     0.00  2025-0909  2025-0910 OPEN  MODET NONE     0.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  23   11        0  1000.00     0.00  2025-0911  2025-0913 OPEN     PO NONE  1000.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  18   11       15 -1000.00     0.00  2025-0913  2025-0914 OPEN  MODET NONE     0.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Inventory Analysis for 0012: W-012,Yoke     0.17 PUR   2  10     0.00    50.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Opening Balance of 0.0 and there are 4 open orders"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  Id Item ParentId  Ordered Assigned      Start   Complete Stat   Type Acti  Balance"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  62   12        0  1000.00     0.00  2025-0830  2025-0909 OPEN     PO NONE  1000.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  42   12       38 -1000.00     0.00  2025-0909  2025-0910 OPEN  MODET NONE     0.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  24   12        0  1000.00     0.00  2025-0903  2025-0913 OPEN     PO NONE  1000.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  19   12       15 -1000.00     0.00  2025-0913  2025-0914 OPEN  MODET NONE     0.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Inventory Analysis for 0013: W-013,Wagon Body     2.25 PUR   2  20     0.00   100.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Opening Balance of 0.0 and there are 2 open orders"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  Id Item ParentId  Ordered Assigned      Start   Complete Stat   Type Acti  Balance"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  63   13        0   500.00     0.00  2025-0802  2025-0822 OPEN     PO NONE   500.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  36   13       35  -500.00     0.00  2025-0822  2025-0901 OPEN  MODET NONE     0.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Inventory Analysis for 0014: W-014,Red Paint    20.00 PUR   2   5     0.00     4.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Opening Balance of 0.0 and there are 2 open orders"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  Id Item ParentId  Ordered Assigned      Start   Complete Stat   Type Acti  Balance"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  64   14        0   125.00     0.00  2025-0817  2025-0822 OPEN     PO NONE   125.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  37   14       35  -125.00     0.00  2025-0822  2025-0901 OPEN  MODET NONE     0.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Inventory Analysis for 0015: W-015,Rear Wheel bracket     1.00 PUR   2   4     0.00    50.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Opening Balance of 0.0 and there are 2 open orders"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  Id Item ParentId  Ordered Assigned      Start   Complete Stat   Type Acti  Balance"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  65   15        0   500.00     0.00  2025-0905  2025-0909 OPEN     PO NONE   500.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  48   15       43  -500.00     0.00  2025-0909  2025-0910 OPEN  MODET NONE     0.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Inventory Analysis for 0016: W-016,Axle Bracket Brace     0.75 PUR   2   6     0.00    50.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Opening Balance of 0.0 and there are 2 open orders"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  Id Item ParentId  Ordered Assigned      Start   Complete Stat   Type Acti  Balance"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  66   16        0   500.00     0.00  2025-0903  2025-0909 OPEN     PO NONE   500.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  49   16       43  -500.00     0.00  2025-0909  2025-0910 OPEN  MODET NONE     0.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Inventory Analysis for 0018: W-018,Carriage Bolt     0.17 PUR   2   3     0.00   500.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Opening Balance of 0.0 and there are 2 open orders"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  Id Item ParentId  Ordered Assigned      Start   Complete Stat   Type Acti  Balance"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  67   18        0   500.00     0.00  2025-0906  2025-0909 OPEN     PO NONE   500.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  50   18       43  -500.00     0.00  2025-0909  2025-0910 OPEN  MODET NONE     0.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Inventory Analysis for 0017: W-017,Rod Stock     0.70 PUR   3   5     0.00  1000.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Opening Balance of 0.0 and there are 15 open orders"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  Id Item ParentId  Ordered Assigned      Start   Complete Stat   Type Acti  Balance"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "   3   17        0  1000.00     0.00  2025-0823  2025-0828 OPEN     PO NONE  1000.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "   2   17        1    -9.00     0.00  2025-0828  2025-0901 OPEN  MODET NONE   991.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  68   17        0  1000.00     0.00  2025-0827  2025-0901 OPEN     PO NONE  1991.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  54   17       53 -1500.00     0.00  2025-0901  2025-0905 OPEN  MODET NONE   491.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  25   17        0  1000.00     0.00  2025-0902  2025-0907 OPEN     PO NONE  1491.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  69   17        0  1000.00     0.00  2025-0902  2025-0907 OPEN     PO NONE  2491.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  21   17       20  -741.00     0.00  2025-0905  2025-0909 OPEN  MODET NONE  1750.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "   5   17        4  -750.00     0.00  2025-0907  2025-0911 OPEN  MODET NONE  1000.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  12   17        0  1259.00     0.00  2025-1002  2025-1007 OPEN     PO NONE  2259.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  26   17        0  1000.00     0.00  2025-1002  2025-1007 OPEN     PO NONE  3259.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "   7   17        6 -1500.00     0.00  2025-1007  2025-1011 OPEN  MODET NONE  1759.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  13   17        0  2250.00     0.00  2025-1102  2025-1107 OPEN     PO NONE  4009.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "   9   17        8 -2250.00     0.00  2025-1107  2025-1111 OPEN  MODET NONE  1759.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  14   17        0  1000.00     0.00  2025-1202  2025-1207 OPEN     PO NONE  2759.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "  11   17       10   -75.00     0.00  2025-1207  2025-1211 OPEN  MODET NONE  2684.00"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : ""
  } ]
}